app/views/admin/events/index.html.haml

```diff
@@ -4,7 +4,10 @@
       %h1
         Events
         = "(#{@events.length})" if @events.any?
-        .btn-group.pull-right
+
+        .pull-right
+          - if can? :create, Event
+            =link_to 'Add Event', new_admin_conference_program_event_path(@conference.short_title), :class => 'button btn btn-default btn-info'
```

Comment:
I would like to discuss it more, my position is:
In comparison to the xls/pdf/csv list export this is the primary action for this page, so it should be visually separated from other actions. Here is some reference http://patternry.com/p=primary-secondary-actions/
If all buttons in green is the standard for the whole project UI - I can make it the regular btn-success
But in general the UI with the clearly separated actions is more usable per many studies. Shouldnt we try to improve the app in this aspect?